dvc/utils/fs.py

```diff
@@ -10,9 +10,11 @@
 
 from dvc.exceptions import DvcException
 from dvc.system import System
+from dvc.utils import _chmod
```

Comment:
Looks like `_chmod()` is only used in `remove()`, should be moved with it. Other things should be too probably, but then `relpath()` is used by by some other things.

Reply:
My logic of not moving the `_chmod` function was that it isn't exactly a 'file-system' utility. It is just being used in one.

Reply:
@algomaster99 `chmod` actually does qualify as a fs utility :) And it is just a helper for `remove` so we shouldn't split it from each other

Reply:
@efiop It just sets write permission for owner right? An fs utility should directly involve with moving or deleting files. Can you clear my confusion?

Reply:
Correct, but `chown` does also qualify as a fs utility, since it operates on paths. But most importantly, it is a mere helper here, that is not used anywhere else, so it doesn't make sense to detach `remove` from `_chmod`.